task-runner/task_runner/task_request_handler.py

```diff
@@ -374,13 +376,29 @@ def _setup_working_dir(self, task_dir_remote) -> str:
             "'_setup_working_dir' called without a task ID.")
 
         task_workdir = os.path.join(self.workdir, self.task_id)
+        sim_workdir = os.path.join(task_workdir, "sim_dir")
```

Comment:
Why is this subdir necessary? Will the simulator find the files?

Reply:
I believe the task runner unzips the input.zip to a folder named "sim_dir". If the files from the bucket are not copied to that subdir the simulator won't find the files and they wont appear in "output.zip".

The order of operation might seem strange, I'm copying the files from the buckets first so that the input.zip files may overwrite any other files.